### DISCUSSION TITLE
你好啊陌生人 (Hello stranger), I'm Hengda 👋🏼 · Hengda Shi 🙊

### DISCUSSION THREAD

**Discussion by giscus** [hengdashi/blog#6] [Announcements]
# 你好啊陌生人 (Hello stranger), I'm Hengda 👋🏼 · Hengda Shi 🙊

I’m a data scientist working for ZipRecruiter on Machine Learning (ML) models for the search engine.

https://hengdashi.dev/me/

**Comment by DerryChan** [comment]
wow, nice blog!

**Reply by hengdashi** [reply]
haha, thanks!
